### DISCUSSION TITLE
What version is stable?

### DISCUSSION THREAD

**Discussion by thisispete** [visgl/luma.gl#1784] [Q&A]
The latest release version listed on the github landing page is 6.1.1 
Docs has a 8.1 version
Digging through the repo I can find up to 8.5 before the 9.0 alphas

**Comment by ibgreen** [comment]
Version 8.5.x is the currently supported version, it is the version used by the latest deck.gl 8.9.

**Comment by ibgreen** [comment]
All releases are documented on https://luma.gl/docs/whats-new/
